src/main/java/no/nav/foreldrepenger/oversikt/aareg/MineArbeidsforholdTjeneste.java

```diff
@@ -0,0 +1,104 @@
+package no.nav.foreldrepenger.oversikt.aareg;
+
+import java.time.LocalDate;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import no.nav.foreldrepenger.common.domain.Fødselsnummer;
+import no.nav.foreldrepenger.oversikt.ereg.VirksomhetTjeneste;
+import no.nav.foreldrepenger.oversikt.saker.PersonOppslagSystem;
+import no.nav.fpsak.tidsserie.LocalDateInterval;
+import no.nav.fpsak.tidsserie.LocalDateSegment;
+
+
+/**
+ * Tjeneste for å hente brukers arbeidsforhold på et gitt tidspunkt fra Aa-registeret.
```

Comment:
Skal denne tjenesten erstatte den som nå er i mottak -api? Altså for å hente informasjon om arbeidsforhold når bruker søker?

Reply:
tenke å samle alle oppslag mot aa-register her på sikt. Vi skal etterhvert konsolidere en del innsendings-apps.